### DISCUSSION TITLE
insert data after migration make errors

### DISCUSSION THREAD

**Discussion by will2318** [prisma/prisma#12647] [Q&A]
Hello, I have a problem that I can't solve.
I have this model:
```prisma
model PermisConduire {
  id             Int     @default(autoincrement()) @id
  noPermis       String   @map("no_permis") @db.VarChar(50)
  dateExpiration DateTime @map("date_expiration") @db.Timestamp(6)

  Chauffeur Chauffeur? @relation("permisConduire_chauffeur")
  @@map("permis_conduire")
}
```
when I migrate the data in my database and then I fill it with an "npx prisma db seed", when adding new users with prismaClient using prisma.permisConduire.create() I get the following error on my unique identifier:
``` return await this.prisma.permisConduire.create
  Unique constraint failed on the fields: (`id`)
```

On the other hand, when I do a migration in my database without doing any seed to add data to my database during the migration, I have no error in my addition of data with prismaClient and the command prisma.permisConduire.create() ?

I don't understand why it behaves like this, can you help me?

thank you all for your answers

**Comment by austincrim** [comment]
Hey @will2318 👋 ,

Could you share your seed script that runs with `npx prisma db seed`?

**Reply by vladi-stevanovic** [reply]
👋 Hello @will2318  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
